feat: new small style corrections, bold for scoreLabel and bit more padding

woile opened the pull request:
Hello, thanks for the extension, it's greate, I made some changes you might be interested.

- contrastRatio preview a bit bigger, to highlight the important part
- score is now bold, I think it helps highlight the contrast a bit better
- added 3.34 as an option for gnome

I wanted to make the inputs smaller but couldn't make it work, I think they take too much "importance" in the extension. I tried modifying the stylesheet adding `height: 10` but didn't work.

Regards

woile committed:
feat: new small style corrections, bold for scoreLabel and bit more padding

larissa commented on contrastRatio.js:
```diff
@@ -38,6 +38,7 @@ const ContrastRatio = new Lang.Class({
 
         this._ratioLabel = new St.Label()
         this._scoreLabel = new St.Label()
+        this._scoreLabel.set_style('font-weight: bold;');
```
Would you mind using a style class instead?

woile replied:
Hey, I don't know how to use style class (barely know how to use gnome, luckly your code is quite self-explanatory), and I couldn't find much documentation, do you mind providing a link to the style class? 
How should it look?
something like this?
```js
const scoreStyle = SomeStyle({"font-weight": "bold"});
this._scoreLabel = new St.Label(scoreStyle)
```

larissa replied:
I believe something like this would work:

```javascript
this._scoreLabel = new St.Label({ reactive: false, style_class: 'your-class-name' })
```

I'm not on linux right now so I can't double check, but let me know if that gives you the results you expect.

larissa commented:
Thanks for sharing your changes and I'm glad you like the extension!

Personally I feel like if the score needs to be bold, it's because the contrast is not good enough in the first place. Maybe an idea would be to make it bold (and bigger) only in cases where the score is AA-Large (or lower), which means the font needs to be 18pt and larger to be readable.

woile commented:
Yes, or make 18px for every case?
I was trying to get a similar result to the mac extension:
![image](https://user-images.githubusercontent.com/2585816/79351248-be8c8e00-7f38-11ea-9ee4-878ba9205ee6.png)

https://usecontrast.com/guide

larissa commented:
Yeah, that sounds good to me. I think in this screenshot the font size and weight looks pretty good creating this line with the other elements and at the same time creating some higher hierarchy for the score. Because on this gnome extension things are not all in the same horizontal line, I think it would look a bit strange to me if the score was that much bigger. But I'm not a designer, so I was just trying to make it functional : P

woile commented:
Okay I've added some changes and I like the result, also added some images to the README. Feel free to merge and change, or reject :+1:  awesome tool!

![image](https://user-images.githubusercontent.com/2585816/79374909-138bcc80-7f58-11ea-82f9-d8f978e56ea5.png)
